### PR TITLE
fxa mono docker image

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Review by dannycoates — APPROVED** [review]
_with glazed eyes_ ... sure why not. Thanks @jbuck!

File: aws/environments/stable3.yml [hunk 1]
```diff
@@ -22,3 +22,5 @@ use_rds: "true"
 
 content_static_resource_url: https://static-stable.dev.lcip.org
 auth_recovery_code_count: 3
+
+fxadev_git_version: fxa-mono-docker-image
```
Comment (jbuck) [inline aws/environments/stable3.yml:26]: Note to self, delete this line before merging

Comment (eviljeff) [reply aws/environments/stable3.yml:26]: 🤔

Comment (jbuck) [reply aws/environments/stable3.yml:26]: I deleted it after merging >.>

https://github.com/mozilla/fxa-dev/commit/8fc5654b1803ef205c502bb77182b0dd9a50341c

**Pull request merged**